packages/components/src/Chip/components/ChipSuffix/Chip.Suffix.test.tsx

```diff
@@ -31,4 +32,77 @@ describe("Chip Compound Components", () => {
     const elem = container.querySelector("." + styles.empty);
     expect(elem).toBeInTheDocument();
   });
+
+  describe("when onClick prop is passed", () => {
+    it("should call onClick when suffix is clicked", () => {
+      const onClick = jest.fn();
+      const { getByTestId } = render(
+        <Chip.Suffix onClick={onClick} testID="ATL-Chip-Suffix">
+          <Icon name="cross" />
+        </Chip.Suffix>,
+      );
+
+      getByTestId("ATL-Chip-Suffix")?.click();
+      expect(onClick).toHaveBeenCalledTimes(1);
+    });
+
+    it("should call onClick when suffix is keypressed", async () => {
+      const onClick = jest.fn();
+      const { getByTestId } = render(
+        <Chip.Suffix onClick={onClick} testID="ATL-Chip-Suffix">
+          <Icon name="cross" />
+        </Chip.Suffix>,
+      );
+
+      const spanElement = getByTestId("ATL-Chip-Suffix");
+      expect(spanElement).not.toBeNull();
+
+      if (spanElement) {
+        spanElement.focus();
+        await userEvent.keyboard("{enter}");
+      }
```

Comment:
Unless I'm missing something, I believe this "if" check is not needed, since there will always be a `spanElement` (at least it should).

Also, instead of `.focus()` you could make use of another user event:
```js
      await userEvent.tab();
      await userEvent.keyboard("{enter}");
      expect(onClick).toHaveBeenCalledTimes(1);
```

Reply:
I can't remember why I put the if statement there 😅 Made that change here f354e43ca9b0fcb6e9a649dd0fd0336f1d1cb942